.github/workflows/ci-host.yml

```diff
@@ -11,12 +11,20 @@ jobs:
     steps:
       - name: Build the container
         run: |
-          DATE=`date "+%F"`
-          IMAGE="ghcr.io/rpm-software-management/dnf-ci-host"
+          # hack: Github replaces secrets with *** in the whole output (even in
+          # e.g. Copr URLs printed by rpm-gitoverlay). If there's a comment (#)
```

Comment:
The mention of Copr URLs is not really relevant in this context, maybe leave it out? Seems like it could just add to confusion instead of being helpful.

Reply:
Ok I have removed that part of the comment.